File: MyApplication/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;
public class MainActivity extends AppCompatActivity {
    private View view;
    public static final String EXTRA_MESSAGE = "com.example.myapplication.MainActivity.MESSAGE";
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
        public void SendMessage(View view){

            Intent i =new Intent(this,MainActivity2.class);

            EditText editText =(EditText) findViewById(R.id.editTextText1);

            String message=editText.getText().toString();

            i.putExtra(EXTRA_MESSAGE,message);

            startActivity(i);
        }
    }

File: MyApplication2/app/src/main/java/com/example/myapplication/MainActivity.kt
 package com.example.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val movies = arrayListOf<String>()

    while (true) {
      println("Enter a movie:  ")
      movies.add(readLine()!!)
      if (movies.size >= 3){
        print("Would you add more movies? (y/n): ")
        if( readLine().toString() == "n")
          break
      }
    }
    println(movies)


    }
}
